disdrodb/l0/check_configs.py

```diff
@@ -357,3 +354,27 @@ def check_raw_array(sensor_name: str) -> None:
 
     if not sorted(list_attributes_L0B_encodings) == sorted(list_attributes_from_raw_data_format):
         raise ValueError(f"Chunksizes in l0b_encodings and raw_data_format for sensor {sensor_name} does not match.")
+
+
+def check_sensor_configs(sensor_name: str) -> None:
+    """check sensor configs.
+
+    Parameters
+    ----------
+    sensor_name : str
+        Name of the sensor.
+    """
+    check_yaml_files_exists(sensor_name)
+    check_variable_consistency(sensor_name)
+    check_l0b_encoding(sensor_name=sensor_name)
+    check_l0a_encoding(sensor_name=sensor_name)
+    check_raw_data_format(sensor_name=sensor_name)
+    check_cf_attributes(sensor_name=sensor_name)
+    check_bin_consistency(sensor_name=sensor_name)
+    check_raw_array(sensor_name=sensor_name)
+
+
+def check_all_sensors_configs() -> None:
```

Comment:
Run this function (or the previous function) in the test routine in order to resolve the reported CodeCoverage warning

Reply:
@You didn't do that right?